### PR TITLE
Added SPELL/ESPELL, spellchecker.

### PR DESCRIPTION
**Pull request by eswenson1**

### CODE REVIEW

**Comment by eswenson1** [comment]
This PR resolves issue #49

**Comment by larsbrinkhoff** [comment]
@eswenson1 If you write just "resolved #49", GitHub will automatically close the issue when the pull request is merged.  See https://github.com/blog/1506-closing-issues-via-pull-requests and https://help.github.com/articles/closing-issues-via-commit-messages/

File: build/build.tcl [hunk 1]
```diff
@@ -664,6 +664,12 @@ expect ":KILL"
 respond "*" ":midas device;atsign rmtdev_gz;rmtdev\r"
 expect ":KILL"
 
+# spell
+respond "*" ":midas sys1;ts spell_syseng;spell\r"
+expect ":KILL"
+respond "*" ":link sys1;ts espell,sys;ts spell\r"
+respond "*" ":link .info.;spell info,info;spell >\r"
```
Comment (larsbrinkhoff) [inline build/build.tcl]: I suppose you added the .INFO. link to mimic PI.  I see the link is used in INFO; DIRLOS 163.  But so far, we don't include the DIRLOS file, and I think it's just an older version.  So maybe that link isn't needed.

**Comment by eswenson1** [comment]
I thought we were trying to mimic PI. I figured that since the link was 
there on PI, and we had the ability to put it there, that I should. But 
I removed it and re-pushed.  Travis is checking the updated build now.  
-- Eric

**Comment by larsbrinkhoff** [comment]
I see there's a file called SPELL; WORDS 212.  Maybe that's needed too.

**Pull request merged**